ui/src/literature/components/__tests__/ExportToCdsModal.test.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import ExportToCdsModal from '../ExportToCdsModal';
+
+describe('ExportToCdsModal', () => {
+  it('renders modal for one paper', () => {
+    const wrapper = shallow(
+      <ExportToCdsModal
+        onOk={jest.fn()}
+        onCancel={jest.fn()}
+        visible={true}
```

Comment:
just for reference, `bool` props can be like:
```suggestion
        visible
```